docs/gcp-professional/services.md

```diff
@@ -272,9 +272,9 @@ Here are some architectures that use AKS as the orchestrator.
 
 | Google Cloud service | Azure service | Description |
 | --- | --- | --- |
-| [Cloud IoT Core](https://cloud.google.com/iot/docs) | [Azure IoT Hub](https://azure.microsoft.com/services/iot-hub/),[Azure Event Hubs](https://azure.microsoft.com/services/event-hubs/) | A cloud gateway for managing bidirectional communication with billions of IoT devices, securely and at scale. |
-| [Cloud Pub/Sub](https://cloud.google.com/pubsub/docs) | [Azure Stream Analytics](https://azure.microsoft.com/services/stream-analytics/#documentation),[HDInsight Kafka](/azure/hdinsight/) | Process and route streaming data to a subsequent processing engine or to a storage or database platform. |
-| [Edge TPU](https://cloud.google.com/edge-tpu) | [Azure IoT Edge](https://azure.microsoft.com/services/iot-edge) | Deploy cloud intelligence directly on IoT devices to run in on-premises scenarios. |
+| [Cloud IoT Core](https://cloud.google.com/iot/docs) <br><b>NOTE:</b> Product has been deprecated by GCP | [Azure Event Grid](https://azure.microsoft.com/services/event-grid/)<br><br> [Azure Event Hubs](https://azure.microsoft.com/services/event-hubs/)<br><br> [Azure IoT Hub](https://azure.microsoft.com/services/iot-hub/) | A cloud gateway for managing bidirectional communication with billions of IoT devices, securely and at scale. |
```

Comment:
The "Description" doesn't seem to cover the two new products you added to this line. Please extend the Description to address those as well.

Reply:
Azure Event Grid and Azure Event Hub is not "A cloud gateway for managing bidirectional communication with billions of IoT devices, securely and at scale."

Reply:
You have three products listed, and only one of them is described by your sentence.

Reply:
With the MQTT updates to Azure Event Grid, it now fits the description. FWIW, I didn't add Azure Event Hubs, only Azure Event Grid - perhaps Event Hubs should be removed.

Reply:
I updated the text and the link, calling out MQTT support specifically.  Thank you.